docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.md

```diff
@@ -13,6 +13,12 @@ related:
     link: docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-method-is-overridden-on-the-project-level.html
 ---
 
+Spryker has more than one thousand modules, and a bunch of them uses in each Spryker-based project. At the same time, the Sprykers team implements about 50 module updates each month. In this case, keeping your project up to date and receiving all necessary updates, coud take a lot of time for the project team.
```

Comment:
```At the same time, the Sprykers team implements about 50 module updates each month```

Here is the focus on ```Sprykers team```, but should be on updates and value, so: 

```
At the same time about 50 Spryker module updates are available for the customers.
```

Reply:
Thanks ++

Reply:
Replaced with the suggested structure